tests/routes/native/v1/authentication_test.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from tests.conftest import TestClient
+from pcapi.core.users.factories import UserFactory
+
+
+def test_send_reset_password_email_without_email(app):
+    response = TestClient(app.test_client()).post("/native/v1/password_reset_request")
+
+    assert response.status_code == 400
+    assert response.json["email"] == ["Ce champ est obligatoire"]
+
+
+def test_request_reset_password_for_unknown_email(app):
+    data = {"email": "not_existing_user@email.com"}
```

Comment:
```suggestion
    data = {"email": "not_existing_user@example.com"}
```
Une bonne pratique est d'écrire les e-mails de test en `example.com` pour éviter d'en envoyer par mégarde.
cf [RFC 2606](https://tools.ietf.org/html/rfc2606).